pyjsx/jsx.py

```diff
@@ -9,6 +11,8 @@
 __all__ = ["jsx"]
 
 
+VALID_KEY_REGEX = re.compile(r"^[a-zA-Z][a-zA-Z0-9_.:-]*$")
```

Comment:
Is this accurate? The spec is a bit different: https://html.spec.whatwg.org/multipage/syntax.html#attributes-2